README.md

```diff
@@ -26,13 +26,13 @@ Examples:
 Component Models:
 RichardsModel <: AbstractSoilModel <: AbstractModel [runnable w/o LandModel wrapper as well]
 
-RootsModel <: AbstractVegetationModel <: AbstractModel  [runnable w/o LandModel wrapper as well]
+VegetationModel <: AbstractVegetationModel <: AbstractModel  [runnable w/o LandModel wrapper as well]
```

Comment:
VegetationModel is probably too general - perhaps PlantHydraulicsModel is better?

Reply:
(there will eventually be many other vegetation components and sub-models)

Reply:
went with SoilPlantHydrologyModel as we brainstormed